src/services/ui/src/features/submission/waiver/capitated/capitated-1915-b-waiver-amendment.tsx

```diff
@@ -84,6 +80,7 @@ const attachmentList = [
 
 export const Capitated1915BWaiverAmendmentPage = () => {
   const location = useLocation();
+  const { id } = useParams("/action/:authority/:id/:type");
```

Comment:
Accessing the new ID (simply `undefined` if navigated to via New Submission flow)